tests/main/ubuntu-core-snapd/task.yaml

```diff
@@ -29,7 +29,10 @@ execute: |
         # we cannot restore in "restore:" because we need a reboot
         # right now to get a clean state again
         systemctl stop snapd.service snapd.socket snapd.autoimport.service snapd.snap-repair.service
```

Comment:
Can you add snapd.snap-repair.timer here too?

Reply:
In a follow-up